sound/soc/sof/intel/hda-stream.c

```diff
@@ -206,37 +214,39 @@ hda_dsp_stream_get(struct snd_sof_dev *sdev, int direction)
 int hda_dsp_stream_put(struct snd_sof_dev *sdev, int direction, int stream_tag)
 {
 	struct hdac_bus *bus = sof_to_bus(sdev);
+	struct sof_intel_hda_stream *hda_stream;
+	struct hdac_ext_stream *stream;
 	struct hdac_stream *s;
-	bool active_capture_stream = false;
+	bool dmi_l1_enable = true;
 	bool found = false;
 
 	spin_lock_irq(&bus->reg_lock);
 
 	/*
 	 * close stream matching the stream tag
-	 * and check if there are any open capture streams.
+	 * and check if there are any capture streams open that are DMI L1 incompatible.
```

Comment:
"any capture streams" -> "any streams" to be consistent with the code, we don't check if it is capture stream here.